bin/dasht-query-line

```diff
@@ -241,6 +253,14 @@ dasht-docsets "$@" | while read -r docset; do
       fragment_by_docset["jQuery"]                  = "#content"
       fragment_by_docset["jQuery_Mobile"]           = "#content"
       fragment_by_docset["jQuery_UI"]               = "#content"
+
+      if (fragments_conf != "") {
+        while (getline < fragments_conf)
```

Comment:
We might avoid reading the fragments file here if it contents are passed in through `-v` option for `awk`.

Reply:
Unfortunately, my `awk` skills are really bad, so I have no idea how to archieve that. Sorry :(